pages/category/index.vue

```diff
@@ -3,10 +3,20 @@
     <SfBreadcrumbs class="breadcrumbs" :breadcrumbs="breadcrumbs" />
     <div class="navbar section">
       <div class="navbar__main">
-        <div class="navbar__aside">
+        <div class="navbar__aside" v-if="!productSearchLoading">
           <SfHeading :level="1" :title="pageHeader" class="category-name sf-heading__title" />
         </div>
-        <div class="navbar__sort">
+        <div class="navbar__aside" v-if="productSearchLoading">
+          <KiboSkeletonLoading
+            class="category-name"
+            skeleton-class="plp-category-name sk-loading"
+          />
+          <KiboSkeletonLoading
+            class="total-products"
```

Comment:
Are we using class="total-products" and class="category-name" with KiboSkeletonLoading? If not then remove it.

Reply:
We are using these classes at component level(category/index.vue).